src/components/Configuration/ConfigDangerArea.vue

```diff
@@ -14,8 +14,8 @@ import { usePollStore } from '../../stores/poll'
 
 import DeletePollDialog from '../Modals/DeletePollDialog.vue'
 import RestorePollIcon from 'vue-material-design-icons/Recycle.vue'
-import ArchivePollIcon from 'vue-material-design-icons/Archive.vue'
-import DeletePollIcon from 'vue-material-design-icons/Delete.vue'
+import ArchivePollIcon from 'vue-material-design-icons/ArchiveOutline.vue'
+import DeletePollIcon from 'vue-material-design-icons/DeleteOutline.vue'
```

Comment:
would be `TrashCanOutline` since we decided to go for Material Symbols, and from the Material Design Icons that would be the matching icon

Reply:
Hey. Let me first finish. :wink:

Reply:
Didn't know you are currently working on it 👍